libs/utils/logWebVitals.js

```diff
@@ -0,0 +1,97 @@
+const LANA_CLIENT_ID = 'pageperf';
+
+let lanaSent = false;
+
+function sendToLana(lanaData) {
+  if (lanaSent) return;
+  const ua = window.navigator.userAgent;
+
+  Object.assign(lanaData, {
+    chromeVer: (ua.match(/Chrome\/(\d+\.\d+\.\d+\.\d+)/) || [])[1] || '',
+    country: sessionStorage.getItem('akamai') || '',
+    // eslint-disable-next-line compat/compat
+    downlink: window.navigator?.connection?.downlink || '',
+    loggedIn: window.adobeIMS?.isSignedInUser() || false,
+    os: (ua.match(/Windows/) && 'win')
+      || (ua.match(/Mac/) && 'mac')
+      || (ua.match(/Android/) && 'android')
+      || (ua.match(/Linux/) && 'linux')
+      || '',
```

Comment:
Nit: this can be less repetitive.
```suggestion
    os: [[/Windows/, 'win'], 
          [/Mac/, 'mac'], 
          [/Android/, 'android'], 
          [/Linux/, 'linux']].find(i => ua.match(i[0]))?.[1] || '';
```

Reply:
I felt Chris version is a little easier to understand on the first read, left it at that